canonical/canonical.com
# WD-9413 - add new /identity page

## Done

Add /identity page and reference it in /diversity page

## QA

- [demo link](https://canonical-com-1213.demos.haus/careers/company-culture/identity)
- [copy doc](https://docs.google.com/document/d/1_-EhhObUF8vl95CypcJEpFcMdvDU4N8ak9kCLvZMtyw/edit)
- [figma](https://www.figma.com/file/RM6wQKCy3u7AyvNJAGQ70v/canonical.com%2Fcareers%2Fdiversity%2Fidentity?node-id=17%3A562&mode=dev)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is done correctly according to figma and copy doc
- Check that the page is referenced on [/diversity](https://canonical-com-1213.demos.haus/careers/company-culture/identity)

## Issue / Card
[WD-9413](https://warthogs.atlassian.net/browse/WD-9413)


[WD-9413]: https://warthogs.atlassian.net/browse/WD-9413?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
